beacon_node/lighthouse_network/src/peer_manager/peerdb.rs

```diff
@@ -290,15 +290,11 @@ impl<E: EthSpec> PeerDB<E> {
     /// Returns a vector of all connected peers sorted by score beginning with the worst scores.
     /// Ties get broken randomly.
     pub fn worst_connected_peers(&self) -> Vec<(&PeerId, &PeerInfo<E>)> {
-        let mut connected = self
-            .peers
+        self.peers
             .iter()
             .filter(|(_, info)| info.is_connected())
-            .collect::<Vec<_>>();
-
-        connected.shuffle(&mut rand::thread_rng());
```

Comment:
Are we intentionally removing the shuffle because the random hashmap ordering is already sufficient to break ties?